adaptive/runner.py

```diff
@@ -128,7 +128,7 @@ def __init__(
 
         self._max_tasks = ntasks
 
-        self._pending_points = {}
+        self._pending_pids = {}
```

Comment:
`pid` is commonly used for a "process id". I propose to adopt the terminology more similar to other parallel executors and call these "_pending_tasks".

Reply:
It's point id here.

Reply:
Indeed, I understand what it means. I should have explained myself clearer: I believe a lot of people when seeing "pid" will think it's a process id, and therefore I propose to rename this attribute.

Reply:
I don't think it will be confusing but nonetheless I've made the suggested change 👍